cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -188,8 +191,7 @@ func (scaleSet *ScaleSet) IncreaseSize(delta int) error {
 // Note that the list results is not used directly because their resource ID format
 // is not consistent with Get results.
 // TODO(feiskyer): use list results directly after the issue fixed in Azure VMSS API.
```

Comment:
Can you please add a GitHub issue link here ? Is it the case that If this issue is fixed then, we don't have to do get but use what comes from the list ?

Reply:
I'm asking them to fix the issue, no Github issue yet.